licenses/Apache-2.0.lre

```diff
@@ -136,13 +145,19 @@ https://opensource.org/licenses/Apache-2.0
       Derivative Works thereof, that is intentionally submitted to Licensor for
       inclusion in the Work by the copyright owner or by an individual or Legal
       Entity authorized to submit on behalf of the copyright owner. For the
-      purposes of this definition, "submitted" means any form of electronic,
-      verbal, or written communication sent to the Licensor or its
-      representatives, including but not limited to communication on
+      purposes of __1__ definition, "submitted" means any form of electronic,
+      verbal, or written
+      ((communication || token))
```

Comment:
Who has token?

Reply:
https://github.com/arsham/retry/blob/master/LICENSE#L54
https://github.com/blokur/harego/blob/master/LICENSE#L54
and other modules by those users.